app/controllers/admin/conferences_controller.rb

```diff
@@ -72,11 +72,15 @@ def index
 
     def new
       @conference = Conference.new
+      @organizations = {}
+      Organization.find_each do |organization|
```

Comment:
I'm not very happy about this, because it will go through EVERY organization in the DB.
Can we do it somehow faster?